source/uwp/Renderer/lib/AdaptiveActionsConfig.cpp

```diff
@@ -11,94 +11,105 @@ namespace AdaptiveCards { namespace Uwp
 {
     HRESULT AdaptiveActionsConfig::RuntimeClassInitialize() noexcept try
     {
-        return S_OK;
+        ActionsConfig actionsConfig;
+        return RuntimeClassInitialize(actionsConfig);
     } CATCH_RETURN;
 
     HRESULT AdaptiveActionsConfig::RuntimeClassInitialize(AdaptiveCards::ActionsConfig actionsConfig) noexcept
     {
-        m_sharedActionsConfig = actionsConfig;
+        m_actionAlignment = static_cast<ABI::AdaptiveCards::Uwp::ActionAlignment>(actionsConfig.actionAlignment);
+        m_actionsOrientation = static_cast<ABI::AdaptiveCards::Uwp::ActionsOrientation> (actionsConfig.actionsOrientation);
+        m_buttonSpacing = actionsConfig.buttonSpacing;
+        m_maxActions = actionsConfig.maxActions;
+        m_spacing = static_cast<ABI::AdaptiveCards::Uwp::Spacing>(actionsConfig.spacing);
+
+        RETURN_IF_FAILED(MakeAndInitialize<AdaptiveShowCardActionConfig>(m_showCardActionConfig.GetAddressOf(), actionsConfig.showCard));
+
         return S_OK;
     }
 
     _Use_decl_annotations_
     HRESULT AdaptiveActionsConfig::get_ShowCard(IAdaptiveShowCardActionConfig** showCardActionConfig)
     {
-        return MakeAndInitialize<AdaptiveShowCardActionConfig>(showCardActionConfig, m_sharedActionsConfig.showCard);
+        ComPtr<IAdaptiveShowCardActionConfig> localShowCardActionConfig = m_showCardActionConfig;
+        *showCardActionConfig = localShowCardActionConfig.Detach();
+        return S_OK;
```

Comment:
We have two patterns throughout the codebase for this:

I believe we should use try to standardize into using CopyTo:

return m_showCardActionConfig.CopyTo(showCardActionConfig)

thoughts?